### ISSUE TITLE
Separate sync and backup notification types (successes separated from failures)

### ISSUE THREAD

**Issue by FibreTTP** [jobobby04/TachiyomiSY#1325]
### Describe your suggested feature

I have syncing enabled with Syncyomi, syncing automatically every time the app is opened, a chapter is opened or read, and every thirty minutes. Each time one of these occurs, a notification is shown.

Since a sync has never failed, I don't know if it would appear in a separate notification channel, but I assume it doesn't, given the name of the channel (simply, "Progress"). This means I cannot disable notifications for sync successes, without also being unaware of when a sync fails.

Presumably, this also applies to backups and restores, given that syncs are in the same notification category; I don't want to give up the ability to know when a backup fails in order to disable the constant "Syncing library complete" notification.

To go further, this should then also apply to backup and restores, having backup successes and failures be separate notification channels.

### Other details

Related: https://github.com/jobobby04/TachiyomiSY/issues/1152
Maybe related: https://github.com/jobobby04/TachiyomiSY/issues/1280

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.11.0](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by wedsa5** [comment]
Came here to suggest this feature request. My phone basically has a persistent Syncyomi notification with how often it happens. I also dont want to completely disable all Syncyomi notifications in case there is a failure.

IMO, the notification categories should be as such:

1. Backup and restore
    1. Allow notifications
    2. Complete
    3. Errors
    4. Progress
2. Sync 
    1. Allow notifications
    2. Complete
    3. Errors
    4. Progress

**Comment by wedsa5** [comment]
I have very little experience with Android dev, but it seems it should be a relatively simple change.

- A few additions to `app/src/main/java/eu/kanade/tachiyomi/data/notification/Notifications.kt` for the new group and channels.
- Then some changes to `app/src/main/java/eu/kanade/tachiyomi/data/sync/SyncNotifier.kt` to use the new channels/group.
- And if a new channel is created for backup errors, an update to `app/src/main/java/eu/kanade/tachiyomi/data/backup/BackupNotifier.kt`

**Comment by lord-ne** [comment]
I don't believe this should be closed. I made it slightly better with #1385, but it still doesn't separate successes from failures, which is what this issue asked for. That's further work (probably not very difficult, but I didn't have the time to look into it)
